drivers/retained_mem/CMakeLists.txt

```diff
@@ -5,5 +5,11 @@ zephyr_syscall_header(${ZEPHYR_BASE}/include/zephyr/drivers/retained_mem.h)
 zephyr_library()
 zephyr_library_sources_ifdef(CONFIG_USERSPACE retained_mem_handlers.c)
 zephyr_library_sources_ifdef(CONFIG_RETAINED_MEM_NRF_GPREGRET retained_mem_nrf_gpregret.c)
+zephyr_library_sources_ifdef(CONFIG_RETAINED_MEM_RV3028       retained_mem_rv3028.c)
 zephyr_library_sources_ifdef(CONFIG_RETAINED_MEM_ZEPHYR_RAM   retained_mem_zephyr_ram.c)
 zephyr_library_sources_ifdef(CONFIG_RETAINED_MEM_ZEPHYR_REG   retained_mem_zephyr_reg.c)
+zephyr_library_sources_ifdef(CONFIG_RETAINED_MEM_RV3028       retained_mem_rv3028.c)
```

Comment:
It seems this file is included twice?

Reply:
I remember noticing that, but I forgot to fix it :))